weblate/trans/checks/chars.py

```diff
@@ -360,3 +360,21 @@ def check_single(self, source, target, unit):
         if self.is_language(unit, ('km', )):
             return False
         return (u'\u200b' in target) != (u'\u200b' in source)
+
+
+class MaxLengthCheck(TargetCheck):
+    '''
+    Check for maximum length of translation.
+    '''
+    check_id = 'max-length'
+    name = _('Maximum Length of Translation')
+    description = _('Translation should not exceed given length')
+    severity = 'danger'
+    default_disabled = True
+    check_value = 0
+
+    def check_target(self, sources, targets, unit):
+        return self.check_target_unit(sources, targets, unit)
```

Comment:
Using parent check_target is better here, or at least you should keep option to ignore the check...